onnxruntime/contrib_ops/rocm/fused_conv.cc

```diff
@@ -15,25 +15,24 @@ namespace {
 // Copied from hipDNN/library/src/hcc_detail/hipdnn_miopen.cpp
 miopenStatus_t _miopenAddTensor(
     miopenHandle_t handle,
-    const void *alpha,
+    const void* alpha,
     const miopenTensorDescriptor_t aDesc,
-    const void *A,
-    const void *beta,
+    const void* A,
+    const void* beta,
     const miopenTensorDescriptor_t cDesc,
-    void *C,
-    const void* zero_scalar)
-{
-    const miopenTensorOp_t tensorOp = miopenTensorOpAdd;
-    // opnd2 = Add ( 0.0 * opnd0, alpha * opnd1 ) + alpha * opnd2
-    return miopenOpTensor(handle, tensorOp,
-                          zero_scalar, cDesc, C,
-                          alpha, aDesc, A,
-                          alpha, cDesc, C);
+    void* C,
+    const void* zero_scalar) {
+  const miopenTensorOp_t tensorOp = miopenTensorOpAdd;
+  // opnd2 = Add ( 0.0 * opnd0, alpha * opnd1 ) + alpha * opnd2
```

Comment:
beta is not used? If so, add some comments.

Reply:
Done.